roles/mariadb/defaults/main.yml

```diff
@@ -1,3 +1,3 @@
 keyserver_fingerprint: "0xcbcb082a1bb943db"
-mirror: nyc2.mirrors.digitalocean.com
+mirror: "ftp.nluug.nl/db"
```

Comment:
I had to change the mirror this morning due to an error, but I think this could be left as it was, it was probably a temporary downtime of the DO's mirror.